pkg/ebpf/gadgets/http/tracer/bpf/http-sniffer.c

```diff
@@ -0,0 +1,623 @@
+#include "sniffer_strcuts.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+
+typedef unsigned short int sa_family_t;
+
+struct
+{
```

Comment:
Why you break line? it's weird. Fix it in all other places as well.

Reply:
DONE